packages/styles/scss/components/locale-modal/_locale-modal.scss

```diff
@@ -267,6 +267,17 @@
     flex: 1;
     overflow-y: auto;
   }
+
+  .#{$prefix}--sr-only {
```

Comment:
This looks like the standard hide for screen readers. @asudoh @kennylam @annawen1 do you know if we have something like this already available either in our repo or Carbon?

Reply:
@jeffchew Yes, there is `.#{$prefix}--assistive-text`.

Reply:
Ah didn't know about that class, will replace it with that one in the next push. Thanks for the heads up!